Task 3

iharmonyharper opened the pull request.

Commit:
do something on develop

Commit:
Task 3

zverok commented on movies.rb:
```diff
@@ -8,15 +8,22 @@
-puts filter(movies, keyword)
+pretty_print(movies.sort_by {|m| m[:duration].to_i}.last(5))
+pretty_print(movies.select {|m| m[:genre][/^comedy$/i]}.sort_by {|m| m[:duration].to_i})
+puts (movies.sort_by {|m| m[:director].split(' ').last}).map {|m| m[:director]}
```
Если здесь местами поменять `map` и `sort`, будет чуть проще.

zverok commented on movies.rb:
```diff
@@ -8,15 +8,22 @@
 
-puts filter(movies, keyword)
+pretty_print(movies.sort_by {|m| m[:duration].to_i}.last(5))
+pretty_print(movies.select {|m| m[:genre][/^comedy$/i]}.sort_by {|m| m[:duration].to_i})
```
`include?` читабельнее чем регексп для этой задачи

zverok commented on movies.rb:
```diff
@@ -8,15 +8,22 @@
+pretty_print(movies.sort_by {|m| m[:duration].to_i}.last(5))
+pretty_print(movies.select {|m| m[:genre][/^comedy$/i]}.sort_by {|m| m[:duration].to_i})
+puts (movies.sort_by {|m| m[:director].split(' ').last}).map {|m| m[:director]}
+puts movies.reject {|m| m[:country][/^usa$/i]}.count
```
`count` может это сделать без дополнительного `reject` ;)
и опять же `include?` более читабельно.

Commit:
Task 3
- #include instead of regexp
- #count instead of #reject+#count
- swap #map and #sort
